src/base/bittorrent/session.cpp

```diff
@@ -321,6 +321,11 @@ QString Session::tempPath() const
     return m_tempPath;
 }
 
+QString Session::torrentTempPath(const InfoHash &hash) const
+{
+    return m_tempPath + QString(hash) + "/";
```

Comment:
No, I wanted to get an explanation, how the presence of the hash of the torrent somewhere in a public place can corrupt the user?